### PR TITLE
Add modals

### PR DESCRIPTION
**Pull request by gigicupp**
Hi @masoncpott can you please code review this?

1. Installed react-icons and imported a "search" icon on the Image.jsx
2. Played with the flex box and now my images and mp4 player displays in the correct layout.
3. Installed react-popup but haven't actually used it yet.

### CODE REVIEW

File: client/src/components/Image.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  padding: 15px;
+  border-radius: 20px;
+`
+const Wrapper = styled.div`
+  /* display: flex; */
+  width: 100%;
+  overflow: hidden;
+`
+
+class Image extends React.Component {
```
Comment (masoncpott) [inline client/src/components/Image.jsx:61]: @gigicupp does this need to be a class based component? If you prefer it, thats fine but I think when a component doesnt explicitly manage its own state the best practice is to use a functional stateless component. I am not positive about that though, something to think about if you have time in the future!

File: client/src/components/Image.jsx [hunk 2]
```diff
@@ -1,24 +1,61 @@
 import React from 'react';
 import styled from 'styled-components';
+import Popup from 'reactjs-popup';
+import { FaSistrix } from 'react-icons/fa';
```
Comment (masoncpott) [inline client/src/components/Image.jsx:4]: maybe i should use this library too!

**Review by masoncpott — APPROVED** [review]

File: client/src/components/Image.jsx (hunk 2 above)
Comment (gigicupp) [reply client/src/components/Image.jsx:4]: @masoncpott 'react-icons' is really cool. Also, the best part is that you can pic and choose which one to import, vs bringing over the entire library. So it helps with your app performance.

File: client/src/components/Image.jsx (hunk 1 above)
Comment (gigicupp) [reply client/src/components/Image.jsx:61]: @masoncpott i may add another child component. so I always start with class and refactor to functional if not needed.

**Pull request merged**